cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -4,6 +4,7 @@ set -e
 
 KUBEVIRT_WITH_ETC_IN_MEMORY=${KUBEVIRT_WITH_ETC_IN_MEMORY:-false}
 KUBEVIRT_WITH_ETC_CAPACITY=${KUBEVIRT_WITH_ETC_CAPACITY:-none}
+DNS_HOST_PORT=${DNS_HOST_PORT:-31111}
```

Comment:
Please add KUBEVIRT prefix for this new variable, it enables distinguishing which env vars are related to kubevirt env.
It also prevents conflicting with other env vars (possibly from other projects), it's not likely to happen but there is a chance.